### PR TITLE
Fix jasmine tests run in random order

### LINKED ISSUE #124: jasmine tests run in random order
Hi!

After update to version 0.3.3 our tests have started to fail unexpectedly.
We have checked why, and found out, that tests were running in a different order each time.

Reverting to 0.3.2 fixed this issue/feature.
(maybe this issue will help someone to figure it out faster)

---

If there was any flag added to the configuration, or something changed - please update plugin description docs.

---
If you need more information about the issue - I will try to give it here.

### PR DESCRIPTION
**Pull request by giridharangm**
Closes #124

### CODE REVIEW

**Comment by BorisOsipov** [comment]
Also have tests for this will be fine
See https://github.com/webdriverio/wdio-jasmine-framework/blob/master/test/adapter.spec.js#L48

**Comment by giridharangm** [comment]
@christian-bromann updated.  
@BorisOsipov Please review the tests.

**Comment by BorisOsipov** [comment]
@giridharangm Awesome, thanks :thumbsup:

File: lib/adapter.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ class JasmineAdapter {
         this.jrunner = new Jasmine()
         let jasmine = this.jrunner.jasmine
 
+        this.jrunner.randomizeTests(!!this.jasmineNodeOpts.random)
```
Comment (christian-bromann) [inline lib/adapter.js]: I guess this line is suppose to be

```js
this.jrunner.randomizeTests(this.getRandomExecutionPolicy())
```

Comment (giridharangm) [reply lib/adapter.js]: Ah! Sorry! my bad. Will fix it now.

**Comment by giridharangm** [comment]
@christian-bromann Done!

**Comment by christian-bromann** [comment]
Released as `v0.3.4`

**Pull request merged**